pom.xml

```diff
@@ -210,7 +211,6 @@
         <dependency>
             <groupId>org.liquibase</groupId>
             <artifactId>liquibase-core</artifactId>
-            <scope>runtime</scope>
```

Comment:
why ?

Reply:
because I need it in the java migration file